modules/ROOT/pages/whats-new-in-studio.adoc

```diff
@@ -1 +1,44 @@
 = What’s New in Studio
+
+Anypoint Studio 7.10 introduces a notification mechanism to help you keep up to date with your modules versions in Exchange, and also enables you to propagate metadata to flows and sub-flows.
+
+== Modules Updates Notifications
```

Comment:
```suggestion
== Module Update Notifications
```
You need not make "module" or "update" plural, because they are adjectives (types) of "updates," which is plural.